CMSIS/DoxyGen/NN/src/history.txt

```diff
@@ -19,6 +19,7 @@
       <li> Added fully_connected int16 reference implementation </li>
       <li> Added fully_connected int16 DSP implementation </li>
       <li> Added unit tests for int16 fully_connected kernel </li>
+      <li> Added dilation support for int8 conv kernel </li>
```

Comment:
Please update the table in https://github.com/ARM-software/CMSIS_5/tree/develop/CMSIS/NN#tfl-micro-compliant-apis as well.

Reply:
done